src/rustup-utils/src/notifications.rs

```diff
@@ -62,12 +58,9 @@ impl<'a> Display for Notification<'a> {
             DownloadFinished => write!(f, "download finished"),
             NoCanonicalPath(path) => write!(f, "could not canonicalize path: '{}'", path.display()),
             ResumingPartialDownload => write!(f, "resuming partial download"),
-            UsingCurl => write!(f, "downloading with curl"),
-            UsingReqwest => write!(f, "downloading with reqwest"),
-            UsingHyperDeprecated => f.write_str(
-                "RUSTUP_USE_HYPER environment variable is deprecated,\
-                 use RUSTUP_USE_REQWEST instead",
-            ),
+            UsingHyperDeprecated => {
+                f.write_str("RUSTUP_USE_HYPER environment variable is deprecated.")
```

Comment:
I don't have a strong feeling either way, but: should there be a warning about `RUSTUP_USE_REQWEST` being deprecated as well, as it is now the only option? If not, should the `RUSTUP_USE_HYPER` warning be removed?

Reply:
Good point, I think you're right that there should be some kind of warning. I'll add that in.